### PR TITLE
Adding new features and fixing issues

### PR DESCRIPTION
**Pull request by ricardobrito**
1. Adding two credit card payment for standard checkout
2. Fixing #14 - notice of undefined index
3. Changing logo extension to png
4. Fixing translation for “Validate”
5. Allowing checkout custom and standard to be enabled together

### CODE REVIEW

File: controllers/front/custompayment.php [hunk 1]
```diff
@@ -120,16 +121,19 @@ private function placeOrder()
...
-				$data['installments'] = $response['installments'];
-				$data['amount'] = Tools::displayPrice($response['amount'], new Currency(Context::getContext()->cart->id_currency), false);
-				$data['payment_id'] = $response['payment_id'];
+				$data['installments'] = array_key_exists('installments', $response) ? $response['installments'] : null;
+				$data['payment_id'] = array_key_exists('payment_id', $response) ? $response['payment_id'] : null;
 				$data['one_step'] = Configuration::get('PS_ORDER_PROCESS_TYPE');
 				$data['valid_user'] = true;
+				
+				$amount = array_key_exists('amount', $response) ? $response['amount'] : 0;
+				$amount = $amount > 0 ? Tools::displayPrice($response['amount'], new Currency(Context::getContext()->cart->id_currency), false) : null;
```
Comment (Quetzacoalt91) [inline controllers/front/custompayment.php]: I guess these two lines could be merged.

Comment (ricardobrito) [inline controllers/front/custompayment.php]: @Quetzacoalt91 
Done

**Comment by Quetzacoalt91** [comment]
Hi @ricardobrito,

Thanks, your PR has just been merged and the changes should be available from tomorrow.
Next time, do not forget to update your config.xml ;)

Best regards

**Comment by ricardobrito** [comment]
Hi @Quetzacoalt91 

Thanks for your quickness!

Regards

**Pull request merged**